packages/ui-motion/src/Transition/props.ts

```diff
@@ -59,15 +59,17 @@ type AllowedPropKeys = Readonly<Array<PropKeys>>
 
 type TransitionProps = TransitionOwnProps & WithStyleProps
 
+const transitionTypes = PropTypes.oneOf([
```

Comment:
I would either call this "TransitionTypePropType", or export only the array of values and use it like `PropTypes.oneOf(transitionTypes)` in the other place(s).

Reply:
fixed